devfile.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
 components:
   - name: tools
     container:
-      image: registry.access.redhat.com/ubi8/openjdk-11 
+      image: registry.redhat.io/codeready-workspaces/plugin-java11-rhel8 
```

Comment:
this image requires authentication

Reply:
this image was used in devfile v1. 
This devfile was created for CRW it shouldn't be a problem with authentication